File: app/src/main/java/com/example/mike/mp3player/commons/PlaybackStateUtil.java
package com.example.mike.mp3player.commons;

import android.os.Bundle;
import android.support.v4.media.session.PlaybackStateCompat;

import static com.example.mike.mp3player.commons.Constants.REPEAT_MODE;

public final class PlaybackStateUtil {

    @Deprecated
    public static Integer getRepeatModeFromPlaybackStateCompat(PlaybackStateCompat playbackStateCompat) {
        if (playbackStateCompat == null || playbackStateCompat.getExtras() == null) {
            return null;
        }
        Bundle extras = playbackStateCompat.getExtras();
        return extras.getInt(REPEAT_MODE);
    }
}

File: commons/src/main/java/com/github/goldy1992/mp3player/commons/PlaybackStateUtil.kt
package com.github.goldy1992.mp3player.commons

import android.support.v4.media.session.PlaybackStateCompat

object PlaybackStateUtil {
    @Deprecated("")
    fun getRepeatModeFromPlaybackStateCompat(playbackStateCompat: PlaybackStateCompat?): Int? {
        if (playbackStateCompat == null || playbackStateCompat.extras == null) {
            return null
        }
        val extras = playbackStateCompat.extras
        return extras!!.getInt(Constants.REPEAT_MODE)
    }
}